nbsite/cmd.py

```diff
@@ -1,7 +1,120 @@
 import os
+import glob
 
 from .util import copy_files
 
 def init(path):
     tmplate = os.path.join(os.path.dirname(__file__),'tmplate')
     copy_files(tmplate,path)
+
+
+git_hosts = {
+    # no trailing slash
+    'GitHub': 'https://raw.githubusercontent.com'
+}
+
+# TODO: really needs cleaning up! Currently just prototyping new behavior.
+def generate_rst(
+        project,
+        examples_path="./examples",
+        doc_path="./doc",
+#        title_formatter=None,
+        git_host='GitHub',
+        git_org=None,        
+        git_repo=None,
+        git_branch='master',
+        # legacy support
+        offset=0,        
+        overwrite=False):
+    """Auto-generates the rst files corresponding to the Notebooks in
+    examples_path
+
+    Takes title from filename, replacing underscores with spaces:
+    The_Title.ipynb -> The Title
+    01_The_Title.ipynb -> 01 The Title
```

Comment:
I think I'd vote for removing leading zeros from anything we show the user, so that the leading zero is *only* in the filename.